### PR TITLE
Update CSI and CPI mirror images list

### PR DESCRIPTION
**Pull request by annablender**
#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in the images-list file
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [x] If updating an existing entry, verify the `SOURCE` is still accurate and upstream hasn't been migrated to a new regitry or repo (if they've migrated, a new repo request to EIO is needed to comply with the `SOURCE DESTINATION TAG` pattern)
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexicographically)
- [x] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

New images
* CSI release driver and syncer: v2.5.4, v2.6.2, v2.7.0
* CPI release manager: v1.24.2, v1.25.0

#### Linked Issues ####

https://github.com/rancher/rancher/issues/38188

#### Final Checks after the PR is merged ####
- [ ] Confirm that you can pull the new images and tags from DockerHub

### CODE REVIEW

**Review by brandond — DISMISSED** [review]

**Comment by brandond** [comment]
Are some of the source tags missing?

File: images-list [hunk 1]
```diff
@@ -126,8 +128,11 @@ gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.5.1
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.5.2
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.5.3
+gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.5.4
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.6.0
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.6.1
+gcr.io/cloud-provider-vsphere/csi/release/syncer rancher/mirrored-cloud-provider-vsphere-csi-release-syncer v2.6.2
+gcr.io/cloud-provider-vsphere/csi/release/syncer rancher/mirrored-cloud-provider-vsphere-csi-release-syncer v2.7.0
```
Comment (jiaqiluo) [inline images-list:135]: Those two lines should be moved to below the line `gcr.io/cloud-provider-vsphere/csi/release/syncer rancher/mirrored-cloud-provider-vsphere-csi-release-syncer v2.5.3`

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Comment by annablender** [comment]
@brandond No, it appears to be a list order issue. csi-release-syncer v2.1.0 exists in the source gcr repo and can be gotten using a `docker pull`

File: images-list (hunk 1 above)
Comment (jiaqiluo) [reply images-list:135]: Update:
 It looks like you are adding the same lines twice in this PR.  Could you remove the duplication?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Pull request merged**